.github/workflows/release.yml

```
@@ -8,16 +8,47 @@
+          token: ${{ secrets.IOS_GITHUB_TOKEN }}
+          sparse-checkout: |
+            Cargo.toml
+          sparse-checkout-cone-mode: false
+      - name: Get rusaint version
+        id: current_version
+        uses: mikefarah/yq@v4
+        with:
+          cmd: yq '.workspace.package.version | "v" + .' Cargo.toml
+      - name: Fetch latest release tag
+        id: latest_release
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:7:18: Double quote to prevent globbing and word splitting

(shellcheck)